src/parser/mod.rs

```diff
@@ -6274,6 +6274,30 @@ impl<'a> Parser<'a> {
             self.expect_keyword(Keyword::WITH)?;
             let table_name = self.parse_object_name(false)?;
             AlterTableOperation::SwapWith { table_name }
+        } else if dialect_of!(self is PostgreSqlDialect)
+            && self.parse_keywords(&[Keyword::OWNER, Keyword::TO])
+        {
+            let next_token = self.next_token();
+            let new_owner = match next_token.token {
+                Token::DoubleQuotedString(ref s) => Owner::Ident(Ident::new(s.to_string())),
```

Comment:
instead of having two separate cases (this DoubleQuotedString and NoKeyword below) to handle the ident variant, would it make sense to call `parse_identifier` if none of the other variants match?
so roughly
```rust
match self.parse_one_of_keywords(&[CURRENT_USER, CURRENT_ROLE, SESSION_USER]) {
    Some(CURRENT_USER) => {}
    ...
    _ =|> self.parse_identifier()
}
```

Reply:
in 0ff17d7